src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -881,7 +889,7 @@ bool AbstractNotationPaintView::ensureViewportInsideScrollableArea()
 bool AbstractNotationPaintView::moveCanvasToPosition(const PointF& logicPos)
 {
     TRACEFUNC;
-
+    LOGW() << "moveCanvasToPosition()";
```

Comment:
Please remove this before we merge the PR (and make sure to return the empty line in its place)